services/api/models/video.pb.go

```diff
@@ -0,0 +1,143 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I'm not sure the protobuf structs can be considered as models. They are contracts but not models I think.